integration-tests/base.ts

```diff
@@ -133,16 +143,19 @@ export async function finalizeTests() {
         }
     }
 
-    console.log(`\n\nsummary:`);
-
     const durationAll = Date.now() - startAll;
 
     if (failureCount === 0) {
-        console.log(green(`  all tests SUCCEEDED in ${durationAll}ms`));
+        console.log(blue('\n\n-------------------- TEARDOWN ------------------------'));
+        await WebServer.shutdown();
```

Comment:
We only want to gracefully shutdown the server if there is no failure?

Reply:
fair enough, we could shutdown before exiting